### PR TITLE
Fix doctor command for android and adb

### PR DESCRIPTION
**Pull request by rosen-vladimirov**
Fix doctor command, which checks adb and android incorrectly. Pass correct paths for verification.
Also make sure "No issues were detected" is printed when there are no issues.

### CODE REVIEW

File: lib/android-tools-info.ts [hunk 1]
```diff
@@ -177,6 +177,24 @@ export class AndroidToolsInfo implements IAndroidToolsInfo {
 		}).future<boolean>()();
 	}
 
+	public getPathToAdbFromAndroidHome(): IFuture<string> {
+		return (() => {
+			if(this.androidHome) {
+				let pathToAdb = path.join(this.androidHome, "platform-tools", "adb") + (this.$hostInfo.isWindows ? ".exe" : "");
```
Comment (teobugslayer) [inline lib/android-tools-info.ts]: Do you really need to take care of the .exe suffix? It should work without it.

**Comment by dtopuzov** [comment]
:100:

File: lib/android-tools-info.ts (hunk 1 above)
Comment (dtopuzov) [inline lib/android-tools-info.ts]: Not sure if it works without, it but it works with .exe and CI is green :) :100:

Comment (rosen-vladimirov) [inline lib/android-tools-info.ts]: yep, I do not need this, I'll remove it (I've added it for debugging purposes and I forgot to remove it).

**Comment by Mitko-Kerezov** [comment]
:+1:

**Comment by teobugslayer** [comment]
Do not forget to squash, :+1:

**Pull request merged**